data/mods/Magiclysm/items/enchanted_belts.json

```diff
@@ -184,14 +182,7 @@
     "color": "brown",
     "covers": [ "torso" ],
     "pocket_data": [
-      {
-        "pocket_type": "CONTAINER",
-        "max_contains_volume": "1 L",
-        "max_contains_weight": "3 kg",
-        "max_item_length": "70 cm",
-        "moves": 2,
-        "holster": true
-      }
+      { "pocket_type": "CONTAINER", "max_contains_volume": "1 L", "max_contains_weight": "3 kg", "moves": 2, "holster": true }
```

Comment:
why did you remove the item length here?

Reply:
didn't realize I did thank you for making me notice